python/TestHarness/TestHarness.py

```diff
@@ -873,15 +522,14 @@ def parseCLArgs(self, argv):
 
         # If attempting to test only failed_tests, open the .failed_tests file and create a list object
         # otherwise, open the failed_tests file object for writing (clobber).
-        self.test_list = []
+        test_list = []
         failed_tests_file = os.path.join(os.getcwd(), '.failed_tests')
         if self.options.failed_tests:
             with open(failed_tests_file, 'r') as tmp_failed_tests:
-                self.test_list = tmp_failed_tests.read().split('\n')
+                self.options._test_list = tmp_failed_tests.read().split('\n')
```

Comment:
The Tester is using this during checkRunnableBase. It works along the same means as --re